apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -75,49 +75,53 @@ export const CreateOrder = () => {
   }
 
   return (
-    <View style={styles.container}>
-      <KeyboardAvoidingView
-        style={styles.flex}
-        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
-        keyboardVerticalOffset={86}
-      >
-        <View style={styles.buttonsGroup}>
-          <ButtonGroup labels={orderTypeLabels} onSelect={handleSelectOrderType} selected={orderTypeIndex} />
-
-          <TouchableOpacity>
-            <Icon.Refresh size={24} />
-          </TouchableOpacity>
-        </View>
+    <>
```

Comment:
it's definitely a View so it doesn't break encapsulation.
```suggestion
    <View>
```